CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guidelines
+If you're interested in contributing to this project, here are a few ways to do so:
+
+## Bug fixes
+* If you find a bug, please first report it using [Github issues](https://github.com/flame-engine/flame/issues/new).
+* Issues that have already been identified as a bug will be labelled "bug".
+* If you'd like to submit a fix for a bug, send a [Pull Request](https://guides.github.com/activities/forking/#making-a-pull-request) from your own fork, also read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
+* Include a test that isolates the bug and verifies that it was fixed.
+* Also update the example and documentation if necessary.
+
+## New Features
+* If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [Github issues](https://github.com/flame-engine/flame/issues/new).
```

Comment:
if it's a bigger change, you can also get some initial thoughts about the general idea on discord beforehand.